extensions/openai/script.py

```diff
@@ -432,9 +442,15 @@ def do_POST(self):
                 self.wfile.write(response.encode('utf-8'))
                 # Finished if streaming.
                 if debug:
+                    if answer and answer[0] == ' ':
+                        answer = answer[1:]
```

Comment:
I think that it's more elegant to use `.lstrip(' ')` in cases like this

Reply:
Hrm, in these cases I explicitly only want to remove a single space, nothing more. other spaces are valid and should remain.

Reply:
fair enough